cfme/services/catalogs/orchestration_template.py

```diff
@@ -1,166 +1,284 @@
-                           context={'template_type': self.template_type,
-                                    'template_name': self.template_name})
-        cfg_btn("Remove this Orchestration Template", invokes_alert=True)
-        sel.handle_alert()
-        flash.assert_success_message('Orchestration Template "{}" was deleted.'.format(
+        view = navigate_to(self, "TemplateDetails")
+        view.configuration.item_select("Remove this Orchestration Template", handle_alert=True)
+        view.flash.assert_message('Orchestration Template "{}" was deleted.'.format(
             self.template_name))
 
     def delete_all_templates(self):
-        sel.force_navigate('orch_template_type',
-                           context={'template_type': self.template_type})
+        view = navigate_to(self, "TemplateType")
         sel.click(pg.check_all())
-        cfg_btn("Remove selected Orchestration Templates", invokes_alert=True)
-        sel.handle_alert()
+        view.configuration.item_select("Remove selected Orchestration Templates", handle_alert=True)
 
     def copy_template(self, template_name, content):
-        sel.force_navigate('select_template',
-                           context={'template_type': self.template_type,
-                                    'template_name': self.template_name})
-        cfg_btn("Copy this Orchestration Template")
-        fill(create_template_form, {'template_name': template_name,
-                                    'content': content},
-             action=create_template_form.add_button)
-        flash.assert_success_message('Orchestration Template "{}" was saved'.format(
+        view = navigate_to(self, "CopyTemplate")
+        view.fill({'name': template_name,
+                   'content': content
+                   })
+        view.add_button.click()
+        view.flash.assert_message('Orchestration Template "{}" was saved'.format(
             template_name))
 
     def create_service_dialog(self, dialog_name):
-        sel.force_navigate('orch_template_type',
-                           context={'template_type': self.template_type})
+        navigate_to(self, "TemplateType")
         if(self.template_type == "CloudFormation Templates"):
             template_name = sel.text("//li[@id='ot_xx-otcfn']/ul"
                 "//a[contains(@class, 'dynatree-title')]").encode("utf-8")
         else:
             template_name = sel.text("//li[@id='ot_xx-othot']/ul"
                 "//a[contains(@class, 'dynatree-title')]").encode("utf-8")
-        sel.force_navigate('create_service_dialog',
-                           context={'template_type': self.template_type,
-                                    'template_name': template_name})
-        fill(dialog_form, {'dialog_name': dialog_name},
-             action=dialog_form.save_button)
-        flash.assert_success_message('Service Dialog "{}" was successfully created'.format(
+        view = navigate_to(self, "AddDialog")
```

Comment:
The `AddDialog` destination is adding from a template, looks like create_service_dialog() and create_service_dialog_from template) are doing the same thing now.